src/TextInput/TextInput.js

```diff
@@ -23,6 +23,9 @@ export default class TextInput extends HTMLElement {
     };
     /** @type {string} */
     #lastChangedValue;
+    // @ts-ignore
```

Comment:
Why ignore?

Reply:
Because in another case, we need to make an unused import of this component. And if we don’t do this, we get an error

Reply:
I don't understand. What kind of error?

Reply:
TS2304: Cannot find name 'InputValidator'

Reply:
Ok, you need to put this explanation whenever you add some "ignore". 

So the problem is that JS Doc doesn't know what type we're referencing without an import - it doesn't know where this type is coming from. But there seems to be a solution to this problem: https://stackoverflow.com/questions/51982814/reference-definition-from-ts-check-js-file